lib/utils.py

```diff
@@ -126,3 +126,14 @@ def run_command(cmd, **kwargs):
                                         stdout=output, stderr=error_output)
 
     return output
+
+
+def replace_str_in_file(file_path, search, replacement):
+
+    lines = []
+    with file(file_path, "r") as f:
+        lines = f.readlines()
+    with file(file_path, "w") as f:
+        for line in lines:
+            line = line.replace(search, replacement)
+            f.write(line)
```

Comment:
What about @mopsfelder's suggestion of using fileinput here? It simplifies this function.